website/docs/r/env.html.markdown

```diff
@@ -0,0 +1,112 @@
+---
+subcategory: "core/v1"
+layout: "kubernetes"
+page_title: "Kubernetes: kubernetes_env"
+description: |-
+  The resource provides mechanisms to inject containers with environment variables 
+  that are managed specifically by Terraform.
+---
+
+# kubernetes_env
+
+The resource provides mechanisms to inject containers with environment variables 
+that are managed specifically by Terraform.
+Environment variables can be used to store fine-grained information such as port numbers and host names.
+
+
+## Example Usage
+
+```hcl
+resource "kubernetes_env" "example" {
+  container = "nginx"
+  metadata {
+    name = "nginx-deployment"
+  }
+
+  api_version = "apps/v1"
+  kind        = "Deployment"
+
+  env {
+    name  = "NGINX_HOST"
+    value = "google.com"
+  }
+
+  env {
+    name  = "NGINX_PORT"
+    value = "90"
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `api_version` - (Required) The apiVersion of the resource to be labelled.
```

Comment:
You have a bunch of copypasta from the `kubernetes_labels` resource. You'll want to replace `resource to be labelled` with `resource to add environment variables to`.